lib/internal/errors.js

```diff
@@ -47,7 +47,6 @@ class AssertionError extends Error {
       throw new exports.TypeError('ERR_INVALID_ARG_TYPE', 'options', 'object');
```

Comment:
I would like to change that as well but this would be a semver major as the error would change again... Therefore: maybe at a later point.